examples/platform/infineon/cyw30739/matter_config.cpp

```diff
@@ -191,8 +192,10 @@ void CYW30739MatterConfig::InitApp(void)
     /* Start CHIP datamodel server */
     static chip::SimpleTestEventTriggerDelegate sTestEventTriggerDelegate{};
     static chip::OTATestEventTriggerHandler sOtaTestEventTriggerHandler{};
+    static Infineon::CYW30739::TestEventTriggerHandler sCYW30739TestEventTriggerHandler{};
     VerifyOrDie(sTestEventTriggerDelegate.Init(chip::ByteSpan(sTestEventTriggerEnableKey)) == CHIP_NO_ERROR);
```

Comment:
Just as an fyi, the `SimpleTestEventTriggerDelegate` isn't a spec compliant implementation of the TestEventTrigger.
The enable key should be in non-volatile storage to avoid needing to recompile the binary after certification since that is prohibited.

Reply:
Thank you for the information.
We will store the enable key as a field of factory data in another change.